examples/todos/main.go

```diff
@@ -130,4 +130,4 @@ func main() {
 
 	log.Println("Serving on " + port)
 	http.ListenAndServe(port, nil)
-}
+}
```

Comment:
Please don't drop empty lines at the end of files, it trips up Git's diff algorithm from time to time and makes staging single-line hunks unreliable. If you don't have it yet, add https://editorconfig.org to your editor and configure it with
```
[*]
end_of_line = lf
insert_final_newline = true
```
for the Jet repository (we can add that config file to the .gitignore if need be).